googletest/googlemock/msvc/2015/gmock.vcxproj

```diff
@@ -103,7 +103,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>..\..\include;..\..;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..\..\..\..\third_party\googletest\googlemock\include;..\..\..\..\third_party\googletest\googlemock\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>

```

Comment:
Should we move this file into the test folder? Maybe something like tests/googletest?

Reply:
These include files are coming from googletest submodule, which is configured in `<repo>\third_party folder. But we can move vsproj files from `<repo>\googletest` to `<repo>\test`. I kept it as it is, so as not to change their references in code.

Reply:
I think we should move them. it would be weird to have random googletest/msvc 2015 files in a folder when it is only intended for tests.

Reply:
Good point. Will move them.